bsdlib/CMakeLists.txt

```diff
@@ -19,10 +19,11 @@ endif()
 # lib oberon are linked in because they are used by the bsd library
 # and must therefore be placed after the bsd library on the linker's
 # command line.
-zephyr_link_libraries(
-  ${BSD_LIB_PATH}/libbsd_nrf9160_xxaa.a
-  ${IMAGE}nrfxlib_crypto
-  c
-  )
+add_library(bsd_nrf9160_xxaa STATIC IMPORTED GLOBAL)
```

Comment:
I don't think this is multi image proof.

My proposal is to do the following:

```
set(libname ${IMAGE}bsd_nrf9160_xxaa)
add_library(${libname} STATIC IMPORTED GLOBAL)
...
```

Reply:
good point.

Let's make it better while we're at it :)